package.json

```
@@ -38,7 +38,7 @@
     "serve": "^11.3.0"
   },
   "engines": {
-    "node": ">= 10.13.0",
+    "node": "^16.0.0",
```

I think we'll want to use
```suggestion
    "node": ">= 16.0.0",
```
here.

`^16.0.0` will mean this package can only be installed on Node v16. In other words: it'll raise an error (or warning, at least) if we install it on a project on Node 18.

But `>= 16.0.0` will mark it as working when Node is _at least_ v16, which is closer to what we want.

(I wish JSON had support for comments so we could document the reasoning here!)

---

I'll make the commit message clear and hopefully that will help explain